Podfile.lock

```diff
@@ -28,9 +28,9 @@ CHECKOUT OPTIONS:
 SPEC CHECKSUMS:
   LetsMove: fefe56bc7bc7fb7d37049e28a14f297961229fc5
   MASShortcut: 9c215e8a8a78f3d01ce56da48e2730ab66b538fa
-  Schedule: 937b2b998a1502e943e37d1b67f38c9e53429c7e
+  Schedule: 0c14f2f813af46b209c3105c2c26600b6195f2b8
   Solar: 2dc6e7cc39186cb0c8228fa08df76fb50c7d8f24
 
 PODFILE CHECKSUM: 8ffe68c48f54016f463965996da2e2d2cdb3b0fe
 
-COCOAPODS: 1.6.1
+COCOAPODS: 1.7.2
```

Comment:
Update your CocoaPods ;)

Reply:
@ApolloZhu Affirmative, but the subject should be you.

Reply:
```
$ pod --version
1.7.5
```